### PR TITLE
added special treatment for Netflix

### LINKED ISSUE #20: Netflix doesn't work
Netflix doesn't work (at least for me). As soon as one user pauses, plays, or changes the video time, Netflix will crash for the other user with error code `M7375`.

It might be possible to circumvent this though using the Netflix player API, instead of directly interacting with the video element (see https://stackoverflow.com/questions/42105028/netflix-video-player-in-chrome-how-to-seek#answer-46816934):

```
const videoPlayer = netflix
  .appContext
  .state
  .playerApp
  .getAPI()
  .videoPlayer

// Getting player id
const playerSessionId = videoPlayer
  .getAllPlayerSessionIds()[0]

const player = videoPlayer
  .getVideoPlayerBySessionId(playerSessionId)
```
and then use 
```
player.play()
player.pause()
player.seek(ms) // timestamp in milliseconds
```
Might try to implement it these days, and if it works I'll make a PR!

### PR DESCRIPTION
**Pull request by nilshellerhoff**
Fixes #20 

I had to add a second content script to inject `netflix.js` into the page header. This was necessary, as content scripts can't normally interact with `window` variables, so `window.netflix` was unavailable. 

Tested in Chrome and Firefox.

### CODE REVIEW

**Comment by nilshellerhoff** [comment]
Nvm, doesn't reliably work with Firefox. Might PR again later

**Comment by nilshellerhoff** [comment]
I have now switched from CustomEvents to window.postMessage and it now also works in Firefox.

File: plugin/js/content.js [hunk 1]
```diff
@@ -130,6 +136,53 @@ function fireEvent(event) {
...
 }
 
+function isNetflix() {
+  // are we on the netflix.com page?
+  return location.host === "www.netflix.com";
+}
+
+function fireEventNetflix(event) {
+  recieved = true;
+  recievedEvent = event.type;
```
Comment (Semro) [inline plugin/js/content.js:146]: It's already in `fireEvent` function. Why not place it on top of  `fireEvent` and remove it from there.

**Review by Semro — CHANGES_REQUESTED** [review]
Clean up all `console.log()`
Lint and formate code with ESLint and Prettier.
Also, it seems you didn't set up git config [user.name](https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git) and [user.email](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address). It's better to set up it for further commits.

**Comment by Semro** [comment]
I don't have Netflix myself to test it, but those changes don't affect other websites and seems Netflix didn't work before, so I merge it.

**Pull request merged**